src/widgets/medications/medication-level-two.test.tsx

```diff
@@ -1,65 +1,119 @@
 import React from "react";
 import { mockPatient } from "../../../__mocks__/patient.mock";
+import { mockFetchPatientMedicationsResponse } from "../../../__mocks__/medication.mock";
 import { cleanup, render, wait } from "@testing-library/react";
 import { BrowserRouter } from "react-router-dom";
-import MedicationLevelTwo from "./medication-level-two.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
 import { of } from "rxjs/internal/observable/of";
-import * as openmrsApi from "@openmrs/esm-api";
+import { fetchPatientMedications } from "./medications.resource";
+import MedicationLevelTwo from "./medication-level-two.component";
 
-describe("<MedicationLevelTwo/>", () => {
-  let patient: fhir.Patient, match;
+const mockUseCurrentPatient = useCurrentPatient as jest.Mock;
+const mockFetchPatientMedications = fetchPatientMedications as jest.Mock;
 
-  afterEach(() => {
-    cleanup();
-  });
+jest.mock("./medications.resource", () => ({
+  fetchPatientMedications: jest.fn()
+}));
+
+jest.mock("@openmrs/esm-api", () => ({
+  useCurrentPatient: jest.fn()
+}));
+
+let wrapper;
+
+describe("<MedicationLevelTwo/>", () => {
```

Comment:
could we use new naming conventions instead of MedicationLevelTwo?

Reply:
Sure